cnames_active.js

```diff
@@ -3099,6 +3099,7 @@ var cnames_active = {
   "ushakov": "ushakov-igor.github.io/ushakov",
   "ushio": "ushio.netlify.com",
   "usv": "usvformat.github.io",
+  "utc": "pythonscratcher.net/utc",
```

Comment:
```suggestion
  "utc": "pythonscratcher.github.io/utc",
```
👋 I suspect you are trying to target https://github.com/PythonScratcher/utc, so you should be using the github.io subdomain for your GitHub account